Sources/PointFree/Bootstrap.swift

```diff
@@ -51,7 +53,12 @@ private let loadEnvVars = { (_: Prelude.Unit) -> EitherIO<Error, Prelude.Unit> i
     .flatMap { try? decoder.decode(EnvVars.self, from: $0) }
     ?? Current.envVars
 
-  update(&Current, \.envVars .~ envVars)
+  Current.envVars = envVars
+  Current.gitHub = GitHub(
```

Comment:
This part is slightly clunky. We manually have to do this work on bootstrap, but at least the side effect is localized...